Tools/PowershellModule/src/Library/Add-WinGetManifest.ps1

```diff
@@ -93,6 +93,7 @@ Function Add-WinGetManifest
         Write-Verbose -Message "Retrieving a copy of the app Manifest file for submission to WinGet source."
         $ApplicationManifest = Get-WinGetManifest -Path $Path
         if(!$ApplicationManifest) {
+            Write-Verbose "$ApplicationManifest`n`n`n"
```

Comment:
Did you mean to have `n`n`n in this?

Reply:
This is intentional. New Line character.